lib/meson.build

```diff
@@ -96,6 +96,7 @@ xnvmelib_deps = [
 xnvmelib_shared = shared_library(
   'xnvme',
   xnvmelib_source,
+  soversion: meson.project_version().split('.')[0],
```

Comment:
If we are going to do it this way - we might as well put `version: meson.project_version()` this will give `libxnvme.so.0` but on linux it will also have `libxnvme.so.0.7.2`

Reply:
Yeah, but it does not fit the Debian conventions of "packages for shared libraries should be named the same as their library" e.g. ``libxnvme0`` then ``lintian`` will expect ``libxnvme.so.0``. If there is then also a ``libxnvme.so.0.7.3`` then ``lintian`` will complain:

https://mentors.debian.net/package/xnvme/

So, I went with only using ``SOVERSION`` encoding in the file-name and then mapping this to ``MAJOR-VER``.

Reply:
Guess i had done some other thing that led me to belief it was not policy-correct. Seems fine now.